src/main/java/org/opentripplanner/standalone/config/RouterConfig.java

```diff
@@ -42,6 +44,9 @@ public RouterConfig(JsonNode node, String source, boolean logUnusedParams) {
         this.streetRoutingTimeoutSeconds = adapter.asDouble(
                 "streetRoutingTimeout", DEFAULT_STREET_ROUTING_TIMEOUT
         );
+        this.bikeRentalServiceDirectoryUrl = adapter.asUri(
+            "bikeRentalServiceDirectoryUrl", null
+        );
```

Comment:
This parameter need documentation in the Configuration.md - Not much, but clearly stated that it is a Sandbox feature.

Reply:
I added a config parameter for the TransmodelAPI today, this how I did it:
Configuration.md:
> `transmodelApi` | configure Entur Transmodel API (**Sandbox**) | object | null | See the code for parameters, no doc provided.
This allow us to add as many parametes as needed later, and the parameter is extracted in the `RouterConfig`like this:
`this.transmodelApiHideFeedId = adapter.path("transmodelApi").asBoolean("hideFeedId", false);`
I didd not create a POJO for the one parameter - i will do that next time I add a new parameter, but the `JSON` structure account for such change.

Reply:
I have added documentation in Configuration.md for this now